<launch>

  <arg name="number_of_robots"  default="1"/>
  <arg name="map_file"          default="/home/daniel/desks.yaml"/>
  <arg name="map_frame"         default="map"/>
  <arg name="slam"              default="false"/>

  <!-- global params -->
  <param name="/number_of_robots" value="$(arg number_of_robots)"/>
  <param name="/scan_range_min"   value="0.45"/>
  <param name="/scan_range_max"   value="5.0"/>
  <param name="/grid_resolution"  value="0.1"/>
  <param name="/pan_goal_tol"     value="0.5"/>
  <param name="/goal_method"      value="frontier"/>

  <!-- exploration visualization -->
  <node pkg="rviz"
        type="rviz"
        name="rviz"
        args="-d $(find csqmi_exploration)/rviz/$(arg number_of_robots)robot.rviz"/>

  <!-- rqt_console for logging
  <node pkg="rqt_console" type="rqt_console" name="rqt_console"/>
 -->
  <!-- map_server -->
  <node unless="$(arg slam)"
        pkg="map_server"
        type="map_server"
        name="map_server"
        output="screen"
        args="$(arg map_file)">
    <param name="frame_id" value="$(arg map_frame)" />
  </node>

</launch>
